plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -150,7 +164,13 @@ const deleteKeyPair = async (sshkeyManager: SshKeyManager) => {
 };
 
 const viewPublicKey = async (sshkeyManager: SshKeyManager) => {
-    const keys: cheApi.ssh.SshPair[] = await sshkeyManager.getAll('vcs');
+    let keys: cheApi.ssh.SshPair[];
+    try {
+        keys = await getKeys(sshkeyManager);
+    } catch (error) {
+        theia.window.showWarningMessage(error.message);
+        return;
+    }
```

Comment:
we could move to a separate method as we're using twice

Reply:
done